web/docker-compose.yml

```diff
@@ -15,9 +15,13 @@ services:
       MYSQL_PASSWORD: superSecretPassword
     ports:
       - "3306:3306"
+    healthcheck:
```

Comment:
używacie depends_on w linii 30, ale żeby to depends_on działało poprawnie, trzeba w jakiś sposób sprawdzić czy baza poprawnie wstała - w tym celu wykonujemy test połączenia zdefiniowany poniżej